rho/scancommand.py

```diff
@@ -324,9 +323,7 @@ def run_ansible_with_vault(cmd_string, vault_pass, ssh_key_passphrase=None,
             if child.isalive():
                 child.wait()
             if log_to_stdout:
-                # tail will kill itself once it is done copying data
-                # to stdout, thanks to the --pid option.
-                tail.wait()
+                tail.terminate()
```

Comment:
That is fine. Seems like an okay compromise. Hopefully we can handle some of the large amounts of output with #154.